frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -2290,6 +2399,68 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                 })
             },
         ],
+        campaignCostsBreakdown: [
+            // this is a temporary query to get the campaign costs breakdown
+            () => [],
+            (): DataTableNode => ({
+                kind: NodeKind.DataTableNode,
+                source: {
+                    kind: NodeKind.HogQLQuery,
+                    query: `
+                        WITH campaign_costs AS (
+                            SELECT 
+                                campaignname,
+                                sum(cost) as total_cost,
+                                sum(clicks) as total_clicks,
+                                sum(impressions) as total_impressions
+                            FROM (
+                                SELECT campaignname, cost, clicks, impressions
+                                FROM bigquery.google_ads_test1.google_ads_2 
+                                WHERE date >= '2025-01-01'
+                                UNION ALL
+                                SELECT campaign_name as campaignname, costinusd as cost, clicks, impressions
+                                FROM bigquery.google_ads_test1.linkedin_ads
+                                WHERE daily >= '2025-01-01'
+                                UNION ALL
+                                SELECT campaignname, spend as cost, clicks, impressions
+                                FROM bigquery.google_ads_test1.bing_ads 
+                                WHERE daily >= '2025-01-01'
+                            )
+                            GROUP BY campaignname
+                        ),
+                        campaign_pageviews AS (
+                            SELECT 
+                                properties.$utm_campaign as campaign_name,
+                                count(*) as pageviews,
+                                uniq(distinct_id) as unique_visitors
+                            FROM events 
+                            WHERE event = '$pageview' 
```

Comment:
Nice, it's easy to see how you could extend this to other conversion goals (like how Brian has both pageview and signup here https://docs.google.com/spreadsheets/d/1JxE2t0C6P9s_5Ee_TTsbNrayNAgwx1kjA9jgqdp3dDw/edit?gid=2007488702#gid=2007488702)